service/discovery/azure/compute.go

```diff
@@ -158,12 +158,18 @@ func (d *azureComputeDiscovery) discoverFunctionsWebApps() ([]voc.IsCloudResourc
 		func(site *armappservice.Site) error {
 			var r voc.IsCompute
 
+			// Get configuration
+			config, err := d.clients.webAppClient.GetConfiguration(context.Background(), *site.Properties.ResourceGroup, *site.Name, &armappservice.WebAppsClientGetConfigurationOptions{})
+			if err != nil {
+				log.Errorf("error getting site config: %v", err)
```

Comment:
You can quickly press the resolve button: Is the following working appropriately if we get error and config is nil/empty... (probably done with tests)

Reply:
Yes, I think it works properly. In handleFunction and handleWebApp we check if the config is empty.